.rubocop.yml

```diff
@@ -9,10 +9,21 @@ AllCops:
 
 # Cop's to ignore
 
+
+# It thinks that Ruby version dependend definitions are duplicates
+Bundler/DuplicatedGem:
+  Exclude:
+    - "Gemfile"
+
 # With this enabled it suggests a change that will break the Gemfile
 Lint/AssignmentInCondition:
   Enabled: false
 
+# It would need a complete rewrite of the test to get this cop satisfied
+Metrics/BlockLength:
+  Exclude:
+    - "spec/**/*.rb"
+
```

Comment:
This cop get's angry about code blocks with more than 25 lines. It would need a complete refactor to satisfy him. The longest block is 1487 lines at the moment.

I think this cop is good for real ruby code but not needed on the spec tests itself.